llvm/lib/LTO/LTOBackend.cpp

```diff
@@ -221,8 +221,17 @@ createTargetMachine(const Config &Conf, const Target *TheTarget, Module &M) {
   else
     CodeModel = M.getCodeModel();
 
+  TargetOptions TargetOpts = Conf.Options;
+
+  if (TargetOpts.MCOptions.ABIName.empty()) {
+    StringRef ModABI = M.getTargetABIFromMD();
+    if (!ModABI.empty()) {
+      TargetOpts.MCOptions.ABIName = ModABI;
+    }
```

Comment:
```suggestion
    TargetOpts.MCOptions.ABIName = M.getTargetABIFromMD();
```
If the result is empty we are just assigning an empty string to an empty string, so we can skip this check?

Reply:
Does not seem to be resolved?

Reply:
seems like my operation miss, let me do that again!